Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -425,8 +405,8 @@ private float maxY
         {
             get
             {
-                int hasMod = (ModuloCheck(NodeList.Count, Tiers) != 0) ? 1 : 0;
-                return NodeList.Count != 0 ? (StepMultiplier(NodeList.Count - (ViewableArea * Tiers), Tiers) + hasMod) * CellHeight : 0.0f;
+                int remainder = (SafeModulo(NodeList.Count, ItemsPerTier) != 0) ? 1 : 0;
```

Comment:
```suggestion
                int partiallyFilledTier = (SafeModulo(NodeList.Count, ItemsPerTier) != 0) ? 1 : 0;
```

Reply:
Or maybe some other better name for it - remainder is a bit misleading because we collapse non zero to zero here (i.e. non zero -> 1, zero -> 0). Really this is trying to find the number of rows at the end that aren't complete because they don't fill an entire thing.

I think that you could probably do the same thing here by just doing a Ceil on SaveDivision

Reply:
Nice one. couldn't find a good name for this :)

Reply:
And Ceil do the job